client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -44,16 +45,24 @@ function getChecklistCompletionProgress( tasks: Task[] | null ) {
 const Sidebar = ( { siteSlug, submit, goNext, goToStep }: SidebarProps ) => {
 	let siteName = '';
 	let topLevelDomain = '';
+	const [ showLaunchTitle, setShowLaunchTitle ] = useState< boolean >( false );
 	const flow = useFlowParam();
 	const translate = useTranslate();
 	const site = useSite();
-	const translatedStrings = getLaunchpadTranslations( flow );
+	const { flowName, title, launchTitle, subtitle } = getLaunchpadTranslations( flow );
 	const arrayOfFilteredTasks: Task[] | null = getArrayOfFilteredTasks( tasks, flow );
 	const enhancedTasks =
 		site && getEnhancedTasks( arrayOfFilteredTasks, siteSlug, site, submit, goToStep );
 
 	const taskCompletionProgress = site && getChecklistCompletionProgress( enhancedTasks );
 
+	useEffect( () => {
+		const launchTask = enhancedTasks?.filter( ( task ) => task.isLaunchTask === true );
```

Comment:
You can simply the conditional check below by using the native find method https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find Ex. `enchancedTasks?.find`

Reply:
Good call. I made that change and it's much cleaner.